src/pages/Admin/Group/Manage/index.jsx

```diff
@@ -0,0 +1,206 @@
+/*
+ Copyright (C) 2022 Samuel Dushimimana (dushsam100@gmail.com)
+
+ SPDX-License-Identifier: GPL-2.0
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+import React, { useEffect, useState } from "react";
+// import messages from "constants/messages";
+
+// Title
+import Title from "components/Title";
+
+// Widgets
+import { Alert, InputContainer } from "components/Widgets";
+
+// Required functions for calling APIs
+import {
+  changeUserPermission,
+  fetchAllGroupMembers,
+  fetchAllGroups,
+} from "services/groups";
+
+import PropTypes from "prop-types";
+import { userPermissions } from "../../../../constants/constants";
+
+const ChangePermissionContainer = ({
+  perm,
+  setShowMessage,
+  setMessage,
+  currGroup,
+  member,
+  handleFetchGroupMembers,
+}) => {
+  const [selectedPerm, setSelectedPerm] = useState(perm);
+
+  useEffect(() => {
+    setSelectedPerm(perm);
+  }, [perm]);
+
+  const handleSetNewPermission = async (newPerm) => {
+    setSelectedPerm(newPerm);
+    try {
+      const res = await changeUserPermission(
+        currGroup,
+        member?.user.id,
+        newPerm
+      );
+      setShowMessage(true);
+      setMessage({
+        type: "success",
+        text: res.message,
+      });
+      handleFetchGroupMembers(currGroup);
+    } catch (e) {
+      setMessage({
+        type: "danger",
+        text: e.message,
+      });
+    } finally {
+      setTimeout(() => {
+        setShowMessage(false);
+      }, [3000]);
+    }
+  };
+  return (
+    <tr>
+      <td>{member?.user.name}</td>
+      <td>
+        <InputContainer
+          type="select"
+          name="name"
+          options={userPermissions}
+          id="select-tag"
+          value={selectedPerm}
+          property="name"
+          onChange={(e) => handleSetNewPermission(e.target.value)}
+        />
+      </td>
+    </tr>
+  );
+};
+
+ChangePermissionContainer.propTypes = {
+  perm: PropTypes.number,
+  setMessage: PropTypes.func,
+  currGroup: PropTypes.number,
+  member: PropTypes.node,
+  handleFetchGroupMembers: PropTypes.func,
+  setShowMessage: PropTypes.func,
+};
+
+const GroupCreate = () => {
```

Comment:
Why we have not created the other file for the GroupCreate function?

Reply:
fixed.